dcos/dcos-ui
# DCOS-49242: [SOAK] TypeError: Cannot read property 'push' of undefined

Display service not found page if the user
tries to edit an unknown servide id.

Closes https://jira.mesosphere.com/browse/DCOS-49242

~This is work in progress, I still have to fix the test.~

## Testing
1. Try to edit a service that doesn't exist.
2. Verify that the UI doesn't break
3. Verify that the "service not found" screen is shown

## Trade-offs
I am not sure what this `props.history` is supposed to do, we don't have it anywhere else in the UI.

## Dependencies
None.

## Screenshots
### Before
![Peek 2019-04-08 14-25](https://user-images.githubusercontent.com/40791275/55728170-3e297800-5a1c-11e9-8400-6d1b284b948c.gif)

### After
![Peek 2019-04-09 15-10](https://user-images.githubusercontent.com/40791275/55801884-b4db7980-5adf-11e9-97f1-b3cbc8a2bd94.gif)
